pages/components/docs/system-props.md

```diff
@@ -11,5 +11,7 @@ To check which system props each component includes, check the documentation for
 |-----|--------|--------|
 | `COMMON`| space, color | [styled-system core docs](https://github.com/jxnblk/styled-system/blob/master/docs/table.md#core) |
 | `TYPOGRAPHY`| fontFamily, fontSize, fontWeight, lineHeight & all `COMMON` props | [styled-system typography docs](https://github.com/jxnblk/styled-system/blob/master/docs/table.md#typography) |
-| `LAYOUT` | borders, borderColor, borderRadius, boxShadow, <br/> display, size, width, height, minWidth, minHeight, <br/> maxWidth, maxHeight, verticalAlign & all `COMMON` props      | [styled-system layout docs](https://github.com/jxnblk/styled-system/blob/master/docs/table.md#layout) <br/> [styled-system misc docs](https://github.com/jxnblk/styled-system/blob/master/docs/table.md#misc) |
+| `LAYOUT` | display, size, width, height, minWidth, minHeight, <br/> maxWidth, maxHeight, verticalAlign & all `COMMON` props      | [styled-system layout docs](https://github.com/jxnblk/styled-system/blob/master/docs/table.md#layout) <br/> [styled-system core docs](https://github.com/jxnblk/styled-system/blob/master/docs/table.md#core) |
```

Comment:
Since the border and box shadow props were removed, I figured we didn't need to link to the `misc` table anymore. Also, the `width` prop is technically documented in the `core` table so I added a link to that table, but I'm not really sure if we need it since the `core` link is already in the `COMMON` row.